content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -38,23 +38,38 @@ https://www.youtube.com/watch?v=szzv4ZXmYAs
      [`mlem init`](https://mlem.ai/doc/command-reference/init), Iterative Studio
      will be able to parse the `.mlem` file to extract model metadata.
 
-4. Provide labels for your model. For example, if your model is about reviewing
+4. If the model is located on the remote storage, Iterative Studio will detect
+   it and suggest to enter path in the repository where to save the model file.
+   In this case, the following will happen:
+
+   - If the repository does not contain DVC, Iterative Studio will run
+     `dvc init`. It is needed to version the model in the git repository.
+     [Learn more](/doc/command-reference/init).
+   - If the specified directory does not exist yet, it will be created.
+   - Iterative Studio will import the model to the repository by executing
+     `dvc import-url <remote_path> <directory_path>/<filename from remote_path> --no-exec`.
+   - Iterative Studio annotate the model by executing
+     `gto annotate <model_name> --path <directory_path>/<filename from remote_path> --type model`.
+     [Learn more](https://github.com/iterative/gto#annotating).
+   - Iterative Studio will create a commit and push it to the repository.
```

Comment:
This part is a bit confusing in general. Esp. for cases where you're adding a model that's already in the selected repo. Users may wonder why anything needs to be committed (since the model is already there). (I realize this comment doesn't apply to the text in question, which is about models imported from external/remote locations, I just wanted to give that feedback -- may be best to make an issue or address separately e.g. in #4070.)

Reply:
Let's not remove this info (that a Git commit is made) as part of this issue; we could add it to https://github.com/iterative/dvc.org/issues/4075 for future improvement.

But here we should definitely explain why a Git commit is needed when the model is already in the Git repo; good point @jorgeorpinel.